### PR TITLE
Add Bootstrap 4 renderer

### PR DESCRIPTION
**Pull request by jtojnar**
Related: https://github.com/nette/forms/pull/209

Closes: #53

### CODE REVIEW

File: src/Rendering/Bs4FormRenderer.php [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * This file is part of the Nextras community extensions of Nette Framework
+ *
+ * @license    New BSD License
```
Comment (hrach) [inline src/Rendering/Bs4FormRenderer.php]: Please remove this line, this repo is MIT and actually the Bs3 probably should be MIT too and definitely new code :)

File: src/Rendering/Bs4FormRenderer.php [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	/** @var Controls\Button */
+	public $primaryButton = null;
+
+	/** @var bool */
+	private $controlsInit = false;
+
+	/** @var int */
+	private $layout = null;
+
+	public function __construct($layout=FormLayout::HORIZONTAL)
```
Comment (hrach) [inline src/Rendering/Bs4FormRenderer.php]: spaces

File: src/Rendering/Bs4FormRenderer.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		}
+
+		$this->controlsInit = true;
+
+		if ($this->layout == FormLayout::INLINE) {
+			$this->form->getElementPrototype()->addClass('form-inline');
+		}
+
+		foreach ($this->form->getControls() as $control) {
+			// TODO: remove after https://github.com/nette/forms/pull/209 is available
```
Comment (hrach) [inline src/Rendering/Bs4FormRenderer.php]: PR is merged :)

Comment (jtojnar) [reply src/Rendering/Bs4FormRenderer.php]: We cannot remove this, until new Nette\Forms is released and we depend on it. I plan to upgrade Nextras\Forms for Nette 3.0 later.

File: src/Rendering/Bs4FormRenderer.php (hunk 2 above)
Comment (jtojnar) [reply src/Rendering/Bs4FormRenderer.php]: Hmm, we should probably incorporate https://github.com/nette/coding-standard into Travis.

**Comment by jtojnar** [comment]
Fixed.

**Comment by jtojnar** [comment]
Ping.

**Comment by hrach** [comment]
Thank you! 🎉

**Comment by vasek-b** [comment]
May I know when you plan to release a new version with the BS4 renderer? Thanks.

**Comment by hrach** [comment]
Soon probably, but if you are looking for Bs3 for Nette 3.0, the repo was split into github.com/forms-rendering

**Pull request merged**